### PR TITLE
[3086][ADD] purchase_order_line_description_adj

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3086](https://www.quartile.co/web#id=3086&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: purchase_order_line_description_adj/models/purchase.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    def _get_product_purchase_description(self, product_lang):
+        display_default_code = self.env.company.po_display_default_code
+        if not display_default_code:
+            self.ensure_one()
+            name = product_lang.with_context(display_default_code=False).display_name
+            if product_lang.description_purchase:
+                name += "\n" + product_lang.description_purchase
+            return name
+        return super()._get_product_purchase_description(product_lang)
```
Comment (yostashiro) [inline purchase_order_line_description_adj/models/purchase.py]:
```suggestion
        product_lang = product_lang.with_context(display_default_code=False)
        return super()._get_product_purchase_description(product_lang)
```
Can't we simply do this without the configuration?

File: purchase_order_line_description_adj/models/purchase.py
Comment (yostashiro) [inline purchase_order_line_description_adj/models/purchase.py:1]: Change the file name to `purchase_order_line.py`.

Class should also be `PurchaseOrderLine`.

**Review by yostashiro — APPROVED** [review]

**Comment by rinaldifirdaus** [comment]
Functional Test Review:

This module has removed the default_code on purchase order line description.
but somehow the alt_code value included with in the description.
discussed with @kanda999 san and confirmed that the alt_code should not included in the description.
related module: [product_alternative_code](https://github.com/qrtl/axls-custom/pull/10)

**Comment by rinaldifirdaus** [comment]
functional test on new commit: LGTM! thanks lin!

**Pull request merged**